pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -216,6 +221,40 @@ func readInstanceID() (string, error) {
 	return md.Uuid, nil
 }
 
+// check opts for OpenStack
+func checkOpenStackOpts(openstackOpts *OpenStack) error {
+	lbOpts := openstackOpts.lbOpts
+
+	// subnet-id is required
+	if len(lbOpts.SubnetId) == 0 {
+		return ErrNoSubnetId
+	}
+
+	// if need to create health monitor for Neutron LB,
+	// monitor-delay, monitor-timeout and monitor-max-retries should be set.
+	emptyDuration := new(MyDuration)
```

Comment:
No need to create a reference to a `MyDuration` only to deref it again later.  Just do: `emptyDuration := MyDuration{}` here and remove the asterisks below.